src/fastga/models/aerodynamics/external/xfoil/xfoil_polar.py

```diff
@@ -140,8 +142,8 @@ def compute(self, inputs, outputs):
         self.options["timeout"] = 15.0
 
         # Get inputs and initialise outputs
-        mach = round(float(inputs["xfoil:mach"]), 4)
-        reynolds = round(float(inputs["xfoil:reynolds"]))
+        mach = round(float(inputs["mach"]), 4)
+        reynolds = round(float(inputs["reynolds"]))
```

Comment:
You can keep the rounding here, but the conversion of 1D array to float will soon be deprecated ^^